README/Text/Pretty.agda

```diff
@@ -0,0 +1,189 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Examples of pretty printing
+------------------------------------------------------------------------
+
+module README.Text.Pretty where
+
+open import Size
+
+open import Data.Bool.Base
+open import Data.List.Base as List
+open import Data.List.NonEmpty as List⁺
+open import Data.Nat.Base
+open import Data.Product
+open import Data.String.Base
+open import Data.Vec.Base as Vec
+open import Function.Base
+
+-- We import the pretty printer and pass 80 to say that we do not want to
+-- have lines longer than 80 characters
+open import Text.Pretty 80
```

Comment:
Will there be other options we'll want to pass in later on? If so is there someway we can future proof this so we don't end up breaking code every time we do so?

Reply:
Jean-Philippe's algorithm only takes one parameter. It is proven correct
(wrt to the 3 criterion he specifies in the article) only for this parameter.
I am not planning to add any other options.